pyrokinetics/constants.py

```diff
@@ -1,15 +1,16 @@
 from scipy import constants
 import numpy as np
+from .units import ureg as units
 
 bk = constants.k
 pi = constants.pi
 mu0 = constants.mu_0
 eps0 = constants.epsilon_0
 
-electron_charge = constants.elementary_charge
+electron_charge = constants.elementary_charge * units.elementary_charge
```

Comment:
I think `units.elementary_charge` is the value + units -- worth double checking at least. I think electron and proton mass are in there too. In fact, we could probably just use those directly?

Reply:
It looks like it is just the value. Not sure scipy supports pint units yet